Python/ceval.c

```diff
@@ -893,16 +907,23 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
     assert(!_PyErr_Occurred(tstate));
 #endif
 
+#ifdef Py_TAIL_CALL_INTERP
+#ifdef LLTRACE
+    return _TAIL_CALL_shim(frame, stack_pointer, tstate, next_instr, 0, 0, lltrace);
```

Comment:
If we move `lltrace` to the frame, we won't need two different forms of `_TAIL_CALL`